### PR TITLE
Add interactive demo for Array.forEach(). Fixes #413

### LINKED ISSUE #413: JS Examples: need example for Array.forEach()
-> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

We should be able to use the existing example-on-top content, ported to interactive-examples. This means converting the content to match the style guide: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md as well as embedding it in an interactive example.

### PR DESCRIPTION
**Pull request by suknuk**
Adding interactive demo for Array.forEach(). Fixes #413 
I am not yet very used to contributing, so any advice or comment is appreciated :)

### CODE REVIEW

File: live-examples/js-examples/array-foreach.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">const array1 = ['a', 'b', 'c'];
```
Comment (wbamberg) [inline live-examples/js-examples/array-foreach.html]: We should declare this with `var` not `const` (see https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6).

File: live-examples/js-examples/array-foreach.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+array1.forEach(function(element) {
+  console.log(element);
+});
+  
+// expected output: "a"
+// expected output: "b"
+// expected output: "c"
+</code>
+</pre>
+
```
Comment (wbamberg) [inline live-examples/js-examples/array-foreach.html]: Please remove the spaces in line 13.

**Comment by suknuk** [comment]
Thank for your review, I have changed the things you requested and amended the commit. Is this how you are supposed to change pull requests?

**Comment by suknuk** [comment]
Ah I see what you mean, thanks for providing the link. I have amended the commit to include a return character at the end of the file

**Comment by wbamberg** [comment]
I've updated the page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach :)

**Pull request merged**